src/libs/ajax/data-table-providers/WdsDataTableProvider.ts

```diff
@@ -177,7 +183,7 @@ export class WdsDataTableProvider implements DataTableProvider {
       supportsExport: false,
       supportsPointCorrection: false,
       supportsFiltering: false,
-      supportsRowSelection: false,
+      supportsRowSelection: true,
```

Comment:
with this set to true, we render the "select all" option in the data table. Selecting that option causes an error for WDS-powered data tables.